reports/report_details.py

```diff
@@ -925,7 +925,8 @@ def add_transnational_to_regions(regions):
     28: 4, 29: 4, 30: 4, 31: 4, 32: 4, 33: 4, 34: 4, 35: 4, 36: 4, 37: 4, 38: 4, 39: 4, 40: 4, 41: 4,
     42: 5, 43: 5, 44: 5, 45: 5, 46: 5, 47: 5, 48: 5,
     49: 6, 50: 6, 51: 6, 52: 6, 53: 6, 54: 6,
-    55: 7
+    55: 7,
```

Comment:
This is due to the new topic added.

Reply:
Which group is this @esirK? Covid?

Reply:
@kilemensi  not really. This was done before I got the info about the new topic **gender & related'** this has been added and should reflect in the code.